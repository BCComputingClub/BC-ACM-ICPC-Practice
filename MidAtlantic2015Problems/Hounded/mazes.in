40 40
XXXXEXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXX XXXXXXXXXX                    XXXXX
XXXX       XXXX XXXXXXXXXXXXXXXXXX XXXXX
XXXXXXXXXX XXXX XXXXXXXXXXXXXXXXXX XXXXX
XXXXXXXXXX XXXX XXXX         XXXXX XXXXX
XXXXXXXXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXXXXXXXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXX   XXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXX K XXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXX   XXXX XXXX XXXX         XXXXX XXXXX
XXXX XXXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXXX XXXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXXX XXXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXXX XXXXX           XXXXXXX XXXXX XXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXX XXXXX XXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXX XXXXX XXXXX
XXXX                         XXXXX XXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXX XXXXX
XXXX                   XXXXX XXXX    XXX
XXXX XXXXXXXXXXXXXXXXX XXXXX XXXX T  XXX
XXXX XXXXXXXXXXXXXXXXX XXXXX XXXX    XXX
XXXX                   XXXXX XXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXX
XXXX                         XXXXXXXXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXX XXXXXXXXXXXXXXXXX         XXXXXXXXX
XXXX XXXXXXXXXXXXXXXXX XXXXXXX XXXXXXXXX
XXXX XXXXXXXXXXXXXXXXX XXXXXXX XXXXXXXXX
XXXX XXXXXXXXXXXXXXXXX XXX XXX XXXXXXXXX
XXXX                   XXX XXX         E
XXXXXXXXXX XXXXXXXXXXXXXXX XXXXXXX XXXXX
XXXXXXXXXX XXXXXX            XXXXX XXXXX
XXXXXXXXXX XXXXXX XXXXXXXXXXXXXXXX XXXXX
XXXXXXXXXX XXXXXX XXXXXXXXXXXXXXXX XXXXX
XXXXXXXXXX XXXXXX XXXXXXXXXXXXXXXX XXXXX
XXXXXXXXXX XXXXXX                  XXXXX
XXXXXXXXXXEXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
40 40
XXXXEXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXX XXXXXXXXXX                    XXXXX
XXXX       XXXX XXXXXXXXXXXXXXXXXX XXXXX
XXXXXXXXXX XXXX XXXXXXXXXXXXXXXXXX XXXXX
XXXXXXXXXX XXXX XXXX         XXXXX XXXXX
XXXXXXXXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXXXXXXXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXX   XXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXX K XXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXX   XXXX XXXX XXXX         XXXXX XXXXX
XXXX XXXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXXX XXXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXXX XXXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXXX XXXXX           XXXXXXX XXXXX XXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXX XXXXX XXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXX XXXXX XXXXX
XXXX                         XXXXX XXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXX XXXXX
XXXX                   XXXXX XXXX    XXX
XXXX XXXXXXXXXXXXXXXXX XXXXX XXXX T  XXX
XXXX XXXXXXXXXXXXXXXXX XXXXX XXXX    XXX
XXXX                   XXXXX XXXX XXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXX XXXXXX
XXXX                         XXXX XXXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXX
XXXX XXXXXXXXXXXXXXXXX         XX XXXXXX
XXXX XXXXXXXXXXXXXXXXX XXXXXXX    XXXXXX
XXXX XXXXXXXXXXXXXXXXX XXXXXXX XXXXXXXXX
XXXX XXXXXXXXXXXXXXXXX XXX XXX XXXXXXXXX
XXXX                   XXX XXX         E
XXXXXXXXXX XXXXXXXXXXXXXXX XXXXXXX XXXXX
XXXXXXXXXX XXXXXX            XXXXX XXXXX
XXXXXXXXXX XXXXXX XXXXXXXXXXXXXXXX XXXXX
XXXXXXXXXX XXXXXX XXXXXXXXXXXXXXXX XXXXX
XXXXXXXXXX XXXXXX XXXXXXXXXXXXXXXX XXXXX
XXXXXXXXXX XXXXXX                  XXXXX
XXXXXXXXXXEXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
40 40
XXXXEXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXX XXXXXXXXXX                    XXXXX
XXXX       XXXX XXXXXXXXXXXXXXXXXX XXXXX
XXXXXXXXXX XXXX XXXXXXXXXXXXXXXXXX XXXXX
XXXXXXXXXX XXXX XXXX         XXXXX XXXXX
XXXXXXXXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXXXXXXXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXX   XXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXX K XXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXX   XXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXXX XXXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXXX XXXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXXX XXXXX XXXX XXXX XXXXXXX XXXXX XXXXX
XXXX XXXXX           XXXXXXX XXXXX XXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXX XXXXX XXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXX XXXXX XXXXX
XXXX                         XXXXX XXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXX XXXXX
XXXX                   XXXXX XXXX    XXX
XXXX XXXXXXXXXXXXXXXXX XXXXX XXXX T  XXX
XXXX XXXXXXXXXXXXXXXXX XXXXX XXXX    XXX
XXXX                   XXXXX XXXX XXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXX XXXXXX
XXXX                         XXXX XXXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXX
XXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXX
XXXX XXXXXXXXXXXXXXXXX         XX XXXXXX
XXXX XXXXXXXXXXXXXXXXX XXXXXXX    XXXXXX
XXXX XXXXXXXXXXXXXXXXX XXXXXXX XXXXXXXXX
XXXX XXXXXXXXXXXXXXXXX XXX XXX XXXXXXXXX
XXXX                   XXX XXX         X
XXXXXXXXXX XXXXXXXXXXXXXXX XXXXXXX XXXXX
XXXXXXXXXX XXXXXX            XXXXX XXXXX
XXXXXXXXXX XXXXXX XXXXXXXXXXXXXXXX XXXXX
XXXXXXXXXX XXXXXX XXXXXXXXXXXXXXXX XXXXX
XXXXXXXXXX XXXXXX XXXXXXXXXXXXXXXX XXXXX
XXXXXXXXXX XXXXXX                  XXXXX
XXXXXXXXXXEXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
40 40
XEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEX
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E          T                           E
E                                      E
E             K                        E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
XEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEX
40 40
XEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEX
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E          T                           E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                     K                E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
E                                      E
XEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEX
0 0
