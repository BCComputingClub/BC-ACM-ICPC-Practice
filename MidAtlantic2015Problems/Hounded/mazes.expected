DROP IT
KEEP IT
KEEP IT
DROP IT
KEEP IT
